libs/openFrameworks/utils/ofUtils.cpp

```diff
@@ -737,7 +737,14 @@ void ofLaunchBrowser(const string& _url, bool uriEncodeQuery){
 //--------------------------------------------------
 string ofGetVersionInfo(){
 	stringstream sstr;
-	sstr << OF_VERSION_MAJOR << "." << OF_VERSION_MINOR << "." << OF_VERSION_PATCH << endl;
+	sstr << OF_VERSION_MAJOR << "." << OF_VERSION_MINOR << "." << OF_VERSION_PATCH;
+
+	if (sizeof(OF_VERSION_PRE_RELEASE) > 1)
```

Comment:
using sizeof here is kind of weird no? something like `string(OF_VERSION_PRE_RELEASE).length()` makes more sense i think